NEWS.md

```diff
@@ -9,6 +9,35 @@ backend.
 
 ## Breaking changes
 
+- The selecting functions (e.g. `gather()` and its `...` argument)
```

Comment:
I think you need to lead with the error message, i.e. " If you see error messages about objects or functions not found, it is because the select functions ..."

And I think you could make it a bit more accessible by starting with something like:

What does the following code do?

```R
x <- 3
df <- tibble(w = 1, x = 2, y = 3)
select(df, 1:x)
```

Does it select the first three columns (using the `x` defined in the global environment), or does it select the first two columns (using the column named `x`)? In this version of tidy we have resolved the ambiguity, now ...